presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -1563,19 +1592,24 @@ public static void writeOrcColumnsPresto(File outputFile, Format format, Compres
 
         writer.write(new Page(blocks));
         writer.close();
-        writer.validate(new FileOrcDataSource(
-                outputFile,
-                new DataSize(1, MEGABYTE),
-                new DataSize(1, MEGABYTE),
-                new DataSize(1, MEGABYTE),
-                true));
+
+        // TODO: sdruzkin - fix row group stats validation for flat maps
+        if (flattenedColumns.isEmpty()) {
```

Comment:
Disabled for files with flat maps because rowGroup stats validation is not implemented for flat map files. It requires quite a lot of work to fix it, so I'll do it in a separate PR to limit the scope of this PR.